src/app/ReadHandler.cpp

```diff
@@ -833,6 +849,12 @@ void ReadHandler::SetDirty(const AttributePathParams & aAttributeChanged)
     {
         InteractionModelEngine::GetInstance()->GetReportingEngine().ScheduleRun();
     }
+    else
+    {
+        // If we couldn't schedule a run because the SetDirty happened before the min Interval, we reschedule the subscription timer
```

Comment:
The comment doesn't match the what I expect in `if (!IsReportable())`, which is what the else would be about.

Also, why are this reschedule and the one below mentioned as "to expire on min interval". Is this the behavior of RescheduleSubscribeTimer?